samples/ComputerVision/AreaOfInterest/Program.cs

```diff
@@ -0,0 +1,87 @@
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                AreaOfInterestSample.RunAsync(endpoint, subscriptionKey).Wait(5000);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+    }
+    public class AreaOfInterestSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(key))
+            {
+                Endpoint = endpoint
+            };
+
+            string localImagePath = @"Images\objects.jpg";   // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/faces.jpg";
+
+            Console.WriteLine("area of interest being found ...");
+            await AreaOfInterestFromUrlAsync(computerVision, remoteImageUrl);
+            await AreaOfInterestFromStreamAsync(computerVision, localImagePath);
+        }
+
+        // Analyze a remote image
+        private static async Task AreaOfInterestFromUrlAsync(ComputerVisionClient computerVision, string imageUrl)
+        {
+            if (!Uri.IsWellFormedUriString(imageUrl, UriKind.Absolute))
+            {
+                Console.WriteLine("\nInvalid remote image url:\n{0} \n", imageUrl);
+                return;
+            }
+
+            AreaOfInterestResult analysis = await computerVision.GetAreaOfInterestAsync(imageUrl);
+
+            Console.WriteLine(imageUrl);
+            DisplayAreas(analysis);
+        }
+
+        // Analyze a local image
+        private static async Task AreaOfInterestFromStreamAsync(ComputerVisionClient computerVision, string imagePath)
+        {
+            if (!File.Exists(imagePath))
+            {
+                Console.WriteLine("\nUnable to open or read local image path:\n{0} \n", imagePath);
+                return;
+            }
+
+            using (Stream imageStream = File.OpenRead(imagePath))
+            {
+                AreaOfInterestResult analysis = await computerVision.GetAreaOfInterestInStreamAsync(imageStream);
+                Console.WriteLine(imagePath);
+                DisplayAreas(analysis);
+            }
+        }
+        private static void DisplayAreas(AreaOfInterestResult analysis)
+        {
+            Console.WriteLine("The Area of Interest is at location {0},{1},{2},{3}",
+                analysis.AreaOfInterest.X, analysis.AreaOfInterest.W,
```

Comment:
We should be consistent with how we display regions in the other samples. Please use X+W and Y+H for the data being printed out.